my_list.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'my_enumerable'
+class MyList
+  include MyEnumerable
+
+  def initialize(list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
```

Comment:
- [ ] You did a great work however when I try the app I have the following error:


<img width="653" alt="Screenshot 2023-10-06 at 2 58 21 PM" src="https://github.com/AjroudiRida/Enumerable/assets/53347966/3e6e3c9f-216c-4f24-99a2-77d781fef8d7">